packages/react-components/react-popover/src/components/PopoverSurface/usePopoverSurface.ts

```diff
@@ -45,10 +46,13 @@ export const usePopoverSurface_unstable = (
       root: 'div',
     },
     root: getNativeElementProps('div', {
+      id,
+      'aria-labelledby': id,
```

Comment:
ah sorry, I think I forgot to mention -- I think the `id` + `aria-labelledby` should go in the examples, rather than in the base component. The reason is that I think having a popover be labelled by all its content is too opinionated as a default. It works well for small popovers like our examples, but not for longer popovers (which should probably be labelled just by the title).

We'll add docs about how to properly label, but for now I think this bit should be in the examples